test/mysql-cdc/30-text-columns.td

```diff
@@ -24,24 +24,29 @@ ALTER SYSTEM SET enable_mysql_source = true
 
 $ mysql-connect name=mysql url=mysql://root@mysql password=${arg.mysql-root-password}
 
+# Insert data into MySQL that can't be decoded using native types and must be decoded
+# as a TEXT COLUMN
+# NOTE: We need to use 'INSERT IGNORE' to have MySQL allow the 0000-00-00 dates which it
+# correctly disallows by default in newer versions, but used to allow in previous ones.
```

Comment:
It might be slightly safer to set `sql_mode = ''`. I think that will allow inserting zero dates in MySQL 8. `INSERT IGNORE` swallows more types of errors than zero dates—e.g., it will silently drop the row if a key constraint is violated, instead of producing an error.

OTOH, this is a test, and so if `INSERT IGNORE` suddenly starts dropping the row entirely, the test will just fail, and that seems plausibly fine...

Reply:
Ah yes good call -- I didn't realize `sql_mode = ''` would effectively do the same thing.